alpha/ui.cpp

```diff
@@ -5,21 +5,21 @@
 // Increase as UI thread uses more things
 #define THREAD_STACK_SIZE 5000
 
-#define DEBOUNCE_DELAY 50
+#define LONG_PRESS_DELAY 1000
+#define LONG_PRESS_CLICK_INTERVAL 500
+#define MIN_SLEEP_DURATION 1000
 
 _TS_UI TS_UI;
 
 // Ctor
 _TS_UI::_TS_UI() {}
 
 // Static function to call instance method
-void _TS_UI::staticTask(void* parameter)
-{
+void _TS_UI::staticTask(void* parameter) {
   TS_UI.task(parameter);
 }
 
-void _TS_UI::begin()
-{
+void _TS_UI::begin() {
```

Comment:
please don't deviate style from rest of code

Reply:
ok, how do i update and include in this PR?

Reply:
ok